src/cryptoadvance/specter/controller.py

```diff
@@ -41,6 +43,22 @@ def inject_debug():
 
 ################ routes ####################
 
+hwi = HWIBridge()
+
+@app.route("/hwi_rpc/", methods=["POST"])
```

Comment:
What do you think about separating the hwi-rpc-stuff to a separate controller-file? The controller is already 700 lines and we should think about reasonable ways to split it up.